recipes/ntp/ntp.inc

```diff
@@ -78,8 +78,11 @@ AUTO_PACKAGE_UTILS = "ntpd ntpdate ntp-wait ntp-keygen sntp ntpq ntpdc \
 RDEPENDS_${PN}-ntpd = "libm"
 RDEPENDS_${PN}-sntp = "libm"
 
-FILES_${PN}-ntpd = "${sysconfdir}/ntp.conf \
-    ${sysconfdir}/init.d/ntpd ${sysconfdir}/rc?.d/???ntpd"
+PACKAGES =+ "${PN}-config ${PN}-init"
```

Comment:
you should probably make some of the other packages pull in these new ones, at least PN